x-pack/plugins/cases/public/components/case_view/index.tsx

```diff
@@ -404,6 +373,18 @@ export const CaseComponent = React.memo<CaseComponentProps>(
       }
     }, [onComponentInitialized]);
 
+    const statusActionButton = useMemo<JSX.Element | null>(
+      () =>
+        caseData.type !== CaseType.collection && userCanCrud ? (
```

Comment:
There is no need to use `useMemo` for components. React will not re-render the component if the props are the same. Also the `StatusActionButton` is exported as `memo(StatusActionButtonComponent)` for that reason.